apps/crc_interactive.py

```diff
@@ -122,7 +124,7 @@ def create_srun_command(self, args: Namespace) -> str:
         srun_dict = {
             'partition': '--partition={}',
             'num_nodes': '--nodes={}',
-            'time': '--time={}:00',
+            'time': '--time={}:00:00',
```

Comment:
This creates `--time=01:00:00:00:00`. The correct form is `'time': '--time={}'`. Time objects automatically format to strings as `HH:MM:SS`.